packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -314,30 +318,47 @@ describe('Ceramic anchoring', () => {
     await ceramic2.close()
   })
 
-  it('test the same state anchored twice, first one wins', async () => {
```

Comment:
As mentioned in my other comment, I still think this test is useful to include and would rather not remove it